src/pages/home/sidebar/OptionRow.js

```diff
@@ -93,43 +103,45 @@ const OptionRow = ({
                             />
                         )
                     }
-                    <View style={[styles.flex1]}>
-                        <Text style={[styles.optionDisplayName, textUnreadStyle]} numberOfLines={1}>
-                            {option.text}
-                        </Text>
-                        {option.alternateText ? (
-                            <Text
-                                style={[textStyle, styles.optionAlternateText, styles.mt1]}
-                                numberOfLines={1}
-                            >
-                                {option.alternateText}
-                            </Text>
-                        ) : null}
-                    </View>
-                    {showSelectedState && (
-                        <View style={[styles.selectCircle]}>
-                            {isSelected && (
-                                <Icon src={Checkmark} fill={themeColors.iconSuccessFill} />
+                            <View style={[styles.flex1]}>
+                                <Text style={[styles.optionDisplayName, textUnreadStyle]} numberOfLines={1}>
+                                    {option.text}
+                                </Text>
+                                {option.alternateText ? (
+                                    <Text
+                                        style={[textStyle, styles.optionAlternateText, styles.mt1]}
+                                        numberOfLines={1}
+                                    >
+                                        {option.alternateText}
+                                    </Text>
+                                ) : null}
+                            </View>
+                            {showSelectedState && (
+                                <View style={[styles.selectCircle]}>
+                                    {isSelected && (
+                                    <Icon src={Checkmark} fill={themeColors.iconSuccessFill} />
+                                    )}
+                                </View>
                             )}
                         </View>
-                    )}
-                </View>
-            </TouchableOpacity>
-            {!hideAdditionalOptionStates && (
-                <View style={styles.flexRow}>
-                    {option.hasDraftComment && (
-                        <View style={styles.ml2}>
-                            <Icon src={Pencil} />
-                        </View>
-                    )}
-                    {option.isPinned && (
-                        <View style={styles.ml2}>
-                            <Icon src={PinCircle} />
+                    </TouchableOpacity>
+                    {!hideAdditionalOptionStates && (
+                        <View style={styles.flexRow}>
+                            {option.hasDraftComment && (
+                            <View style={styles.ml2}>
```

Comment:
Looks like this `<View>` and its children should be indented by four more spaces.